public/css/content.css

```diff
@@ -13,7 +13,7 @@
   padding: .5em 1em; border-left: 5px solid silver;
   background-color: #f0f0f0;
 }
-#content table { border-spacing: 0; }
+#content table { border-spacing: 0; table-layout: fixed;}
```

Comment:
If it's not set, CodeMirror expands horizontally if you exceed the content instead of showing scrollbar. Verified.